build-logic/kotlin-convention/build.gradle.kts

```diff
@@ -0,0 +1,23 @@
+plugins {
+    `kotlin-dsl`
+    id("com.avito.android.libraries")
```

Comment:
The grouping or maybe just naming is not clear for me.

- Directory: `kotlin-convention` - expect to see what Kotlin needs
- Plugin id: `.libraries` - kinda "all libraries"?
- Dependencies: kotlin, AGP?, buildlogic

How about this?

```kotlin
id("<>.kotlin")

dependencies {
    implementation("com.avito.android.buildlogic:libraries")
    implementation(platform("com.avito.android.infra:platform"))
    implementation(libs.kotlinPlugin)
    // without AGP
}
```